cpp/ql/src/semmle/code/cpp/models/implementations/Memcpy.qll

```diff
@@ -13,61 +13,73 @@ import semmle.code.cpp.models.interfaces.Taint
  * The standard functions `memcpy` and `memmove`, and the gcc variant
```

Comment:
This comment needs an update to include `bcopy`.

Reply:
Done.